Essentials/src/com/earth2me/essentials/commands/Commandrepair.java

```diff
@@ -38,7 +39,7 @@ public void run(final Server server, final User user, final String commandLabel,
     }
 
     public void repairHand(User user) throws Exception {
-        final ItemStack item = user.getBase().getItemInHand();
+        final ItemStack item = user.getBase().getInventory().getItemInMainHand();
```

Comment:
Off hand was added in 1.9 so this will break on any versions before then (such as the popular 1.8.8). Do we want to add a version check for this? Or are we breaking <1.9 support completely?

Reply:
Seems like `/repair`, `/more` and `/itemdb` were updated to do this in one of the initial 1.13 commits (1a820ad9b7d2564d836f3044f770aec56c0b04a1). We could implement a method that tries `getItemInMainHand` and falls back to `getItemInHand`?